dynaconf/utils/parse_conf.py

```diff
@@ -276,6 +276,20 @@ def get_converter(converter_key, value, box_settings):
     return converted_value
 
 
+def add_converter(converter_key, casting_callable):
+    global converters
+
+    def converter_callable(value, box_settings=None):
```

Comment:
We can use functools.wraps to make this generated function to inherit name and docstring from the passed function. https://docs.python.org/3/library/functools.html#functools.wraps

```suggestion
    @functools.wraps(casting_callable)
    def converter_callable(value, box_settings=None):
```